File: solutions/uri/1015/1015.py
from math import pow, sqrt

x1, y1 = map(float, input().split())
x2, y2 = map(float, input().split())

distance = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))

print(f"{distance:.4f}")

File: solutions/uri/1015/1015.clj
(ns main
  (:require [clojure.string :as str]))

(defn str->double
  [sequence]
  (map #(Double/parseDouble %) sequence))

(defn read-line->double []
  (-> (read-line)
    (str/split #" ")
    (str->double)))

(defn main []
  (let [[x1 y1] (read-line->double)
        [x2 y2] (read-line->double)]
    (printf
      "%.4f%n"
      (Math/sqrt (+ (Math/pow (- x2 x1) 2) (Math/pow (- y2 y1) 2))))))

(main)
